docs/antrea-ipam.md

```diff
@@ -46,3 +46,145 @@ cluster. Valid range is 16 to 30. Default is 24.
 
 - `nodeCIDRMaskSizeIPv6`: Mask size for IPv6 Node CIDR in IPv6 or dual-stack
 cluster. Valid range is 64 to 126. Default is 64.
+
+## Antrea Flexible-IPAM
+
+Since version 1.4, Antrea starts to support flexible control over Pod IP addressing. This
+can be achieved by configuring custom resource `IPPool` with desired set of IP ranges.
```

Comment:
Probably say: `IPPool` CRD

a desired set of

Reply:
Fixed.